packages/block-library/src/text-columns/save.js

```diff
@@ -20,7 +15,7 @@ export default function save( { attributes } ) {
 				<div className="wp-block-column" key={ `column-${ index }` }>
 					<RichText.Content
 						tagName="p"
-						value={ get( content, [ index, 'children' ] ) }
+						value={ content?.[ index ]?.children }
```

Comment:
We're being flexible here since the `content` comes from an outside attribute and could potentially not exist.